pkg/stanza/adapter/benchmark_test.go

```diff
@@ -6,6 +6,7 @@ package adapter
 import (
 	"context"
 	"fmt"
+	"go.uber.org/zap"
```

Comment:
This will have to be moved to the second group of imports in order to pass the linter